### DISCUSSION TITLE
Issue output.css

### DISCUSSION THREAD

**Discussion by Snaily0O** [tailwindlabs/tailwindcss#13811] [Help]
![image_2024-06-08_203305932](https://github.com/tailwindlabs/tailwindcss/assets/114316682/3c2012c0-dc98-4b72-bae9-73d3b9caabe1)
![image_2024-06-08_203331226](https://github.com/tailwindlabs/tailwindcss/assets/114316682/d0c12030-42b9-4e12-9a17-328fda1e85b8)

![image_2024-06-08_203414668](https://github.com/tailwindlabs/tailwindcss/assets/114316682/6e951081-a84d-425b-8898-ef6e1b3012bb)

Hello, i don't understand why i have this error when i start the index.html 
Can someone help me please?

**Comment by wongjn** [comment]
What are the steps you are performing to cause the error?

**Reply by Snaily0O** [reply]
https://www.youtube.com/watch?v=h4nI6thys6E&ab_channel=HeroCode

**Reply by Snaily0O** [reply]
look the description

**Reply by Snaily0O** [reply]
I did exactly the same thing

**Reply by wongjn** [reply]
Which step in the list of steps causes the error to occur?

**Reply by Snaily0O** [reply]
this is not a step, after making the command 'npm run watch' i start the index.html and I have this error (the style on the page does not work)

![image_2024-06-08_204713113](https://github.com/tailwindlabs/tailwindcss/assets/114316682/7b0f3519-f00d-4350-bb25-ce3d8f8c04d5)

**Reply by wongjn** [reply]
Consider checking you have linked the CSS file properly in the `index.html`. You can confirm this by checking if you have a `404` response for the CSS file in the *Network* tab in the browser's developer tools.

**Reply by Snaily0O** [reply]
![image_2024-06-08_205057302](https://github.com/tailwindlabs/tailwindcss/assets/114316682/4c6d7b1f-558e-4bf4-bbd0-24c81dbe8474)
![image_2024-06-08_205115042](https://github.com/tailwindlabs/tailwindcss/assets/114316682/94b60160-6747-4a84-a643-824437129c0c)
![image_2024-06-08_205124130](https://github.com/tailwindlabs/tailwindcss/assets/114316682/1535a61a-d928-4f20-a1a6-1b56f1e42958)

**Reply by Snaily0O** [reply]
it's weird.. I don't understand

**Reply by wongjn** [reply]
Ah OK, so you are incorrectly linking the CSS file as a JavaScript file. Instead, consider linking it as a stylesheet like:

```diff
-<script src="output.css"></script>
+<link rel="stylesheet" href="output.css"/>
```

**Comment by Snaily0O** [comment]
problem solved sorry, it's  ' link rel="stylesheet" href="./output.css" '

**Reply by Snaily0O** [reply]
thanks you wongjn
